components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -152,23 +151,102 @@ The default policies are: Viewers, Editors, Administrator, and Ingest.
+Without a policy statement to allow an operation tagged with a given project, a user cannot perform the actions of that policy.
+Note that Chef Automate is currently limited to six projects while we continue to refine the user experience during this beta period.
+
+There are four steps to setting up projects:
+
+1. Define each project via the UI (or the command line if you prefer).
+   Start on the projects list page shown previously to create your projects.
+2. Create or modify IAM policy statements to restrict permissions to specific projects as needed.
+   *Every* statement must either indicate specific projects or specify the wildcard denoting all projects (`*`).
+   By default, any previously existing policies are automatically setup with that wildcard so they apply to all projects.
+3. Tag resources with projects (see [Tagging Resources]({{< relref "iam-v2-overview.md#tagging-resources" >}})).
+4. Select the projects to filter in the UI.
+   Once you create some projects, use the **global project filter** in the penultimate position of the top navigation bar to select one or more projects for viewing.
+   With no selection (the default), you are thereby saying you want *no* filtering, hence displaying resources for *all* projects for which you have permission.
+
+Note that out-of-the-box we provide a *project-owner* role so the global admin may delegate much of these responsibilities for project management to others, to alleviate some of the burden for the global admin.
+
+### Properties of a Project
+
+A v2.1 project consists of a list of **rules**, where each rule describes a group of node characteristics.
+
+Property   | Description
+-----------|------------
+Rules      | list of tagging rules
+
+A rule consists of a list of **conditions**, where each condition describes a single characteristic.
+It also contains a type where <<TODO describe what node and event mean>>
+
+Property   | Description
+-----------|------------
+Type       | one of (node, event)
+Conditions | list of node characteristics
+
+A condition consists of these properties:
+
+Property   | Description
+-----------|------------
+Attribute  | for **event** types: one of (CHEF_ORGANIZATION, CHEF_SERVER)
+Attribute  | for **node** types: one of (CHEF_ORGANIZATION, CHEF_SERVER, ENVIRONMENT, CHEF_ROLE, CHEF_TAG, CHEF_POLICY_NAME, CHEF_POLICY_GROUP)
+Operator   | one of (equals, member_of)
+Values     | list of one or more values to match on the specified attribute
+
+### Tagging Resources
+
+There are two broad categories of resources that may be tagged: your set of nodes feeding data into Chef Automate and the set of IAM resources themselves.
+Any of these that you do not explicitly tag (as described next) are considered *unassigned* with respect to projects.
+We denote this group of unassigned projects as being in the `(unassigned)` project, including the parentheses as a way to connote that it is a special designator.
```

Comment:
users will not have the same understanding of _(unassigned)_ as we do, since it will not show up in the projects list. so please, can we not refere to it as the unassigned project, it is instead just resources that have not been assigned to a project. happy to talk more about this when im back if youd like to understand more of my reasoning.